src/templates/subscription/bar.html

```diff
@@ -0,0 +1,9 @@
+<div class="pf-widget-body">
+</div>
+<a class="pf-widget-close">&times;</a>
+<div class="pf-bar-content">
+	<p class="pf-widget-message"></p>
+	<form>
+		<input name="email" type="email" required><input type="submit" class="pf-widget-btn pf-widget-ok" />
```

Comment:
totally agree, hopefully having these in a readable format will help us recognize some of these things. was thinking basically:

* get all the existing merges, minus this one merged
* add new templates to this pr and merge
* make a pass at existing templates to make them all consistent
* tackle mobile styles

more or less